web-elements/src/test/java/com/demoqa/TextBoxFormTest.java

```diff
@@ -0,0 +1,87 @@
+package com.demoqa;
+
+import com.demoqa.page.TextBoxPage;
+import org.junit.jupiter.api.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TextBoxFormTest extends BaseTest {
+
+    @Test
+    public void testPageIsOpenTest(){
+        TextBoxPage textBoxPage=new TextBoxPage(driver);
+        textBoxPage.open("text-box");
+        String currentURL = driver.getCurrentUrl();
+        String expectedURL = "https://demoqa.com/text-box";
+        assertEquals(currentURL,expectedURL);
+    }
+    @Test
+    public void fullNameTest(){
+        TextBoxPage textBoxPage=new TextBoxPage(driver);
+        textBoxPage.open("text-box");
+        textBoxPage.enterFullName("Aleksandra");
+        String actualResult=textBoxPage.getTextFullName();
+        String expectedResult="Aleksandra";
+        assertEquals(expectedResult,actualResult);
+    }
+    @Test
+    public void userEmailTest(){
+        TextBoxPage textBoxPage=new TextBoxPage(driver);
+        textBoxPage.open("text-box");
+        textBoxPage.enterUserEmail("aleksandra@gmail.com");
+        String actualResult=textBoxPage.getTextUserEmail();
+        String expectedResult="aleksandra@gmail.com";
+        assertEquals(expectedResult,actualResult);
+    }
+    @Test
+    public void currentAddressTest(){
+        TextBoxPage textBoxPage=new TextBoxPage(driver);
+        textBoxPage.open("text-box");
+        textBoxPage.enterCurrentAddress("Dnipro,prospect Svobody,3");
+        String actualResult=textBoxPage.getTextCurrentAddress();
+        String expectedResult="Dnipro,prospect Svobody,3";
+        assertEquals(expectedResult,actualResult);
+    }
+    @Test
+    public void permanentAddressTest(){
+        TextBoxPage textBoxPage=new TextBoxPage(driver);
+        textBoxPage.open("text-box");
+        textBoxPage.enterPermanentAddress("Dnipro,prospect Svobody,3");
+        String actualResult=textBoxPage.getTextPermanentAddress();
+        String expectedResult="Dnipro,prospect Svobody,3";
+        assertEquals(expectedResult,actualResult);
+    }
+    @Test
+    public void formTextBoxTest(){
+        TextBoxPage textBoxPage=new TextBoxPage(driver);
+        textBoxPage.open("text-box");
+        textBoxPage.enterFullName("Aleksandra");
```

Comment:
Use prepared data.
Firstly create variables such as 'String fullName, String email' etc.
and after using these variables in the test

Each test should have 3 blocks:
given - incoming data
when - actions that we do in a test
then - checks expected result with actual